.github/workflows/preview.yml

```diff
@@ -0,0 +1,35 @@
+name: Deploy to Preview
+
+on:
+  push:
+    branches: [main]
```

Comment:
Did you copy this from the Welsh Petitions repo? If so the default branch name is `main` there so this will need to be `master` to work.

Reply:
Aha, I thought [main] might be a github keyword for the primary branch. Updated.